charts/incubator/tt-rss/values.yaml

```diff
@@ -179,3 +187,80 @@ service:
         targetSelector: main
         port: 9000
         targetPort: 9000
+configmap:
+  config:
+    enabled: true
+    data:
+      nginx.conf.template: |-
+        worker_processes auto;
+        pid /var/run/nginx.pid;
+
+        events {
+            worker_connections  1024;
+        }
+
+        http {
+          include /etc/nginx/mime.types;
+          default_type  application/octet-stream;
+
+          access_log /dev/stdout;
+          error_log /dev/stderr warn;
+
+          sendfile on;
+
+          index index.php;
+
+          # Removed during app integration to Truecharts.
+          # nginx's internal resolver skips the search domains present in /etc/resolv.conf
+          # See:
+          # nginx resolver requires fqdn: https://stackoverflow.com/a/40695219
+          # variable in proxy_pass forces use of resolver: https://serverfault.com/a/593003
+          # tt-rss discussion: https://community.tt-rss.org/t/resolving-issues-with-latest-commit-on-k8s/6208/7
+          # proxy_pass and resolver: https://stackoverflow.com/a/22259088
+          #resolver ${RESOLVER} valid=5s;
+
+          server {
+            listen 80;
+            listen [::]:80;
```

Comment:
Lets remove ipv6 and change 80 to `{{ .Values.service.main.ports.main.port }}`

Then remove the `targetPort: 80` completely!